content/blog/cosign-2.5.0.md

```diff
@@ -0,0 +1,132 @@
+
+Harbor v2.5 integrates support for [Cosign](https://github.com/sigstore/cosign), a OCI artifact signing and verification solution that is part of the [Sigstore project](https://github.com/sigstore).
+
+Cosign signs OCI artifacts and pushes the generated signature into Harbor. This signature is stored as an artifact accessory along side the signed artifact. Harbor manages a link between the signed artifact and cosign signature, allowing you to apply things like **tag retention rules** and **immutable rules** to a signed artifact, and it will extend to both the signed artifact and the signature. In this way you can use Harbor's built in functionality to manage signed artifacts and Cosign signature accessories.
+
+A key feature of using Cosign with Harbor is the ability use Harbor's [replication capabilities](https://goharbor.io/docs/2.5.0/administration/configuring-replication/) to replicate signatures with their associated signed artifact. This means that if a **replication rule** applies to a signed artifact, Harbor will apply the replication rule to the signature in the same way it applies it to the signed artifact.
+
+* When replicating between Harbor instances, the target Harbor instance will maintain the link between the signed artifact and its associated signatures. You will be able to see the relationship between the two artifacts in the target Harbor interface, in the same way that you do in the source registry.
+
+Full Documentation you will be able to find [here](https://goharbor.io/docs/2.5.0/working-with-projects/working-with-images/sign-images/)
+
+## Demo Setup 
+**Configure two Harbor instances with Cosign enabled per repo and configure replication**
+
+#### 1. Install two instances with cosig(and notary) enabled
+   In our setup we use [offline installer](https://goharbor.io/docs/2.5.0/install-config/download-installer/).
+
+```
+# ./install.sh --with-notary --with-trivy
+
+[Step 0]: checking if docker is installed ...
+...
+...
+...
+[Step 5]: starting Harbor ...
+Creating network "harbor_harbor" with the default driver
+Creating network "harbor_harbor-notary" with the default driver
+Creating network "harbor_notary-sig" with the default driver
+Creating harbor-log ... done
+Creating harbor-portal ... done
+Creating redis         ... done
+Creating registryctl   ... done
+Creating registry      ... done
+Creating harbor-db     ... done
+Creating trivy-adapter ... done
+Creating notary-signer     ... done
+Creating harbor-core   ... done
+Creating harbor-jobservice ... done
+Creating nginx             ... done
+Creating notary-server     ... done
+✔ ----Harbor has been installed and started successfully.----
+```
+
+#### 2. Create new project "cosign"
+![Cosign Project](../img/cosign/cosign-project.png)
```

Comment:
the path of the images in this docx seems not correct? 

It should be
`![Cosign Project](./img/cosign/cosign-project.png) `

rather than
`![Cosign Project](../img/cosign/cosign-project.png)`

Reply:
thought so as well but the site rendering is changing it! 

example here : https://goharbor.io/blog/harbor-extending-its-reach/
https://goharbor.io/blog/img/machine-learning.png

hugo creates a directory for that blog post!

Reply:
yap, forget about it.